assets/runtime/functions

```diff
@@ -819,23 +797,15 @@ initialize_datadir() {
     exit 1
   fi
 
-  # create dotfiles directory
-  mkdir -p ${REDMINE_DOTFILES_DIR}
-  chmod -R u+rw,go+r ${REDMINE_DOTFILES_DIR}
-  chown -R ${REDMINE_USER}: ${REDMINE_DOTFILES_DIR}
-
-  mkdir -p ${REDMINE_DOTFILES_DIR}/.ssh
-  chmod -R u+rw,go-rwx ${REDMINE_DOTFILES_DIR}/.ssh
-  chown -R ${REDMINE_USER}: ${REDMINE_DOTFILES_DIR}/.ssh
-
-  mkdir -p ${REDMINE_DOTFILES_DIR}/.subversion
-  chmod -R u+rw,go+r ${REDMINE_DOTFILES_DIR}/.subversion
-  chown -R ${REDMINE_USER}: ${REDMINE_DOTFILES_DIR}/.subversion
-
-  # create tmp directory
-  mkdir -p ${REDMINE_DATA_DIR}/tmp
-  chmod -R u+rw,go+r ${REDMINE_DATA_DIR}/tmp
-  chown -R ${REDMINE_USER}: ${REDMINE_DATA_DIR}/tmp
+  # fix permissions for relevant folders
+  datafolders="${REDMINE_DATA_DIR} ${REDMINE_PLUGINS_DIR} ${REDMINE_THEMES_DIR} ${REDMINE_ATTACHMENTS_DIR} ${REDMINE_BACKUPS_DIR} ${REDMINE_DOTFILES_DIR} ${REDMINE_DOTFILES_DIR}/.ssh ${REDMINE_DOTFILES_DIR}/.subversion ${REDMINE_DATA_DIR}/tmp"
```

Comment:
Note the .ssh directory gets different permissions.  It should be handled outside of this loop.

Reply:
fix in updated code. I'll keep the code for .ssh  as is and avoid the find wrapper. As of the low number of files, there might be no benefit at all.